boards/arc/em_starterkit/support/openocd.cfg

```diff
@@ -1,6 +1,12 @@
 # Configure JTAG cable
 # EM Starter Kit has built-in FT2232 chip, which is similar to Digilent HS-1.
-source [find interface/ftdi/digilent-hs1.cfg]
+interface ftdi
```

Comment:
If you're updating this to accommodate newer OpenOCD please make sure you follow suggested renames (`interface` -> `adapter driver`) etc. Look at OpenOCD's debug messages where all that is explained.

Note, other options might need update as well.